tests/e2e/ctl_v3_elect_test.go

```diff
@@ -74,8 +75,9 @@ func testElect(cx ctlCtx) {
 	if err = blocked.Signal(os.Interrupt); err != nil {
 		cx.t.Fatal(err)
 	}
-	if err = e2e.CloseWithTimeout(blocked, time.Second); err != nil {
-		cx.t.Fatal(err)
+	err = e2e.CloseWithTimeout(blocked, time.Second)
+	if err != nil {
+		require.ErrorContains(cx.t, err, "unexpected exit code")
```

Comment:
@serathius that's another candidate:
```
[unexpected exit code [1] after running [/home/tjungblu/git/etcd/bin/etcdctl --endpoints=http://localhost:20000 --dial-timeout=7s elect a p2]]
```

Reply:
actually `CloseWithTimeout` also looks a little more difficult than it needs to be.